wiki/Ranking_Criteria/osu!mania/zh.md

```diff
@@ -101,7 +97,7 @@ outdated_since: 17ba49c633b089fe88acb7d5d32ac0460e5d65fe
 
 #### 准则
 
-- **回血速度 (HP) 和判定严度 (OD) 不应高于 7。**
+- **回血限度 (HP) 和判定严度 (OD) 不应高于 7。**
```

Comment:
[来源请求]，下同

Reply:
因为这两个游戏模式的 HP 影响的不仅仅是回复的速度，也影响惩罚的量度。它更像个限制而不是速率。
讨论了一下，选择保持之前的版本。
because HP in these game modes affects not only the speed of recovery but also the severity of punishment. It looks like a *limitation* rather than *speed*. 
after the discussion, I chose to keep the previous translation.